app/renderer/components/preferences/paymentsTab.js

```diff
@@ -365,8 +369,10 @@ const styles = StyleSheet.create({
   // History and settings icons
   switchWrap__mainIconsRight: {
     position: 'relative',
-    right: '12px',
-    top: '3.5px'
+    top: '3.5px',
```

Comment:
I know it was like this before does but this extra `.5px` makes any difference?

Reply:
I remember I used a screen ruler and adding it made it look nicer a bit.